test-app/src/common/utils.ts

```diff
@@ -6,6 +6,7 @@ import { StacksMainnet, StacksTestnet } from '@stacks/network';
 dayjs.extend(relativeTime);
 
 const testnetUrl = 'https://stacks-node-api.testnet.stacks.co';
+const mainnetUrl = 'https://stacks-node-api.stacks.co';
```

Comment:
Just changed this testing locally so will revert before merging. Having trouble today testing testnet with the test-app.